services/venice-controller/src/main/java/com/linkedin/venice/controller/kafka/consumer/AdminExecutionTask.java

```diff
@@ -426,8 +426,6 @@ private void handleSetStoreCurrentVersion(SetStoreCurrentVersion message) {
     String storeName = message.storeName.toString();
     int version = message.currentVersion;
     admin.setStoreCurrentVersion(clusterName, storeName, version);
-
-    LOGGER.info("Set store: {} version to {} in cluster: {}", storeName, version, clusterName);
```

Comment:
why do we want to remove this?

Reply:
This line moved to `VeniceHelixAdmin#setStoreCurrentVersion` - and will be logged even when there is single-region mode.